src/rust/engine/src/rule_graph.rs

```diff
@@ -794,6 +890,11 @@ impl RuleEdges {
     }
     RuleEdges { dependencies: new_deps, selector_to_dependencies: new_selector_deps }
   }
+
+  pub fn entries_for(&self, selectors: &Vec<Selector>) -> Entries {
```

Comment:
Rather than taking a `&Vec<T>`, recommend taking a `&[T]` (ie, a "slice")... the former can be viewed as the latter, as can a static array literal like `entries_for(&[1, 2])`.